test-suite/coq-makefile/coqdoc1/run.sh

```diff
@@ -9,7 +9,13 @@ make html mlihtml
 make install DSTROOT="$PWD/tmp"
 make install-doc DSTROOT="$PWD/tmp"
 #make debug
-(for d in `find tmp -name user-contrib` ; do pushd $d >/dev/null && find . && popd >/dev/null; done) | sort -u > actual
+(
+  while IFS= read -r -d '' d
+  do
+    pushd "$d" >/dev/null && find . && popd >/dev/null
+  done <   <(find tmp -name user-contrib)
```

Comment:
Shouldn't you use `find -print0` if you use `read -d ''`? Same for other copies of this pattern.